website/description_issues.qmd

```diff
@@ -3,20 +3,25 @@ title: "DESCRIPTION file Issues"
 editor: visual
 ---
 
-# Issue 1
+# Formatting Software Names
 
 ## Problem
 
-## Solution
+The automatic spell check has flagged a software name as incorrect and results in a NOTE.
 
-# Issue 2
+## Solution
 
-## Problem
+All references to names of software, packages and API names in the description section of the DESCRIPTION file should be wrapped in single quotes. e.g.: 'ggplot2', 'python', etc.
 
-## Solution
+The description section in the [DESCRIPTION file](https://github.com/tidyverse/ggplot2/blob/main/DESCRIPTION) of the 'ggplot2' package provides an example:
 
-# Issue 3
+```         
+Description: A system for 'declaratively' creating graphics, based on "The
+    Grammar of Graphics". You provide the data, tell 'ggplot2' how to map
+    variables to aesthetics, what graphical primitives to use, and it
+    takes care of the details. 
+```
 
-## Problem
+### Details
 
-## Solution
+Words in single quotes are not flagged by the automatic spell check and may be used when appropriate to include references to software names. It may also be useful and necessary to document non-obvious acronyms which can produce a NOTE in the [cran-comments.md](https://usethis.r-lib.org/reference/use_cran_comments.html) file.
```

Comment:
Maybe we should add a paragraph telling people nit to use single quotes to circumvent the spell check. And that _**only**_ software, package and API names should have single quotes (and not function names, acronymes, uncommon words flagged as wrong in the spell check etc...). 

Maybe we can also find a different example, as the one you provided does exactly this and wraps 'declaratively' in single quotes.

Reply:
There was a solution some time ago posted on r-devel or r-pkg-devel that marked some words as accepted to avoid being flagged by checks. This solution would also be helpful for folks that have "weird" words in other parts of the package.